app/ui/window.js

```diff
@@ -41,6 +41,14 @@ module.exports = class Window {
       window.setBackgroundColor(toElectronBackgroundColor(cfg_.backgroundColor || '#000'));
     };
 
+    // set startup directory
+    let workingDirectory = cfgDir;
```

Comment:
You have to move this block below, in handler passed to `app.config.subcribe()` and after config decoration by plugins.
This way, your working directory config will be hot reloaded and modifiable by plugins.

Bonus: please find a way to get rid of this use of `process.argv` in this module. (Already here before your PR, I know 😁 ) 🙏